exercises/complex-numbers/test/Tests.hs

```diff
@@ -0,0 +1,191 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Prelude hiding    (abs, div)
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import ComplexNumbers
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = do
+          describe "real"      $ for_ realCases      $ testC real
+          describe "imaginary" $ for_ imaginaryCases $ testC imaginary
+          describe "conjugate" $ for_ conjugateCases $ testB conjugate
+          describe "abs"       $ for_ absCases       $ testC abs
+          describe "mul"       $ for_ mulCases       $ testA mul
+          describe "div"       $ for_ divCases       $ testA div
+          describe "add"       $ for_ addCases       $ testA add
+          describe "sub"       $ for_ subCases       $ testA sub
+ where
+  testA f CaseA{..} = it descriptionA $ f number1A number2A `shouldBe` expectedA
+  testB f CaseB{..} = it descriptionB $ f number1B `shouldBe` expectedB
+  testC f CaseC{..} = it descriptionC $ f number1C `shouldBe` expectedC
+
+
+data CaseA = CaseA { descriptionA :: String
```

Comment:
Possible to have more descriptive names than A, B, C?

BinOp, UnOp, Scalar?

Reply:
Could, be. Don't know if it is worth with something this simple.